static_src/test/functional/overview_quicklook.spec.js

```diff
@@ -0,0 +1,50 @@
+
+import './global_setup';
+
+import OrgQuicklookElement from './pageobjects/org_quicklook.element';
+
+
+describe('Overview page', function () {
+  it('navigates to page', function () {
+    browser.url('/');
+  });
+
+  it('has a title', function () {
+    expect(browser.getTitle()).toBe('cloud.gov dashboard');
+  });
+
+  it('has a page header', function () {
+    const pageHeader = browser.element('.page-header-title');
```

Comment:
Okay, so in this case, would I add an additional class, `.test-page-header-title` to the page header?